src/pmlogger/pmlogger.service.in

```diff
@@ -5,6 +5,7 @@ After=network-online.target pmcd.service
 Before=pmlogger_check.timer pmlogger_daily.timer
 BindsTo=pmlogger_check.timer pmlogger_daily.timer
 Wants=pmcd.service
+Wants=pmlogger_farm.service
```

Comment:
To follow up on our discussion, there is a WantedBy=pmlogger.service line in pmlogger_farm.service which should have (?) served this purpose - this new Wants= line creates a circular dependency between the two services which does not seem correct.